packages/flutter/test/material/tabs_test.dart

```diff
@@ -254,6 +254,14 @@ void main() {
     expect(tester.getSize(find.byType(Tab)), const Size(14.0, 72.0));
   }, skip: isBrowser);
 
+  testWidgets('Tab sizing - icon, iconMargin and text', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      MaterialApp(theme: ThemeData(fontFamily: 'Ahem'), home: const Center(child: Material(child: Tab(icon: SizedBox(width: 10.0, height: 10.0), iconMargin: EdgeInsets.symmetric(horizontal: 100.0), text: 'x')))),
```

Comment:
Can you add line breaks so that each parameter is on its own line for readability?

Reply:
Updated in the new commit with each parameter being on its own line. 😄Let me know if you've more feedback.

Only kept them in a single line to be consistent with the rest of the tests because they were all in a single line.

Reply:
Oh sorry, I wouldn't have mentioned it if I saw that all the other tests are the same way.  Anyway better to move in the direction of matching the styleguide for now.

One test failed but it's just a timeout, I'll restart it and then merge when it goes green.  Thanks for the quick response!

Reply:
Thanks. All checks passed now! 👏🏻